mirrord/config/src/feature/env.rs

```diff
@@ -111,6 +113,26 @@ pub struct EnvConfig {
     /// These variables will override environment fetched from the remote target.
     #[config(env = MIRRORD_OVERRIDE_ENV_FILE_ENV)]
     pub env_file: Option<PathBuf>,
+
+    /// ### feature.env.mapping {#feature-env-mapping}
+    ///
+    /// Specify map of patterns that if matched will replace the value according to specification.
+    ///
+    /// *Capture groups are allowed.*
+    ///
+    /// Example:
+    /// ```json
+    /// {
+    ///   ".+_TIMEOUT": "10000"
+    ///   "LOG_.+_VERBOSITY": "debug"
+    /// }
+    /// ```
+    ///
+    /// Will do the next replacements for environment variables that match:
+    ///
+    /// `CONNECTION_TIMEOUT: 500` => `CONNECTION_TIMEOUT: 10000`
+    /// `LOG_FILE_VERBOSITY: info` => `LOG_FILE_VERBOSITY: debug`
+    pub mapping: Option<HashMap<String, String>>,
```

Comment:
We need to verify regexes during config validation

Reply:
Added to the `verify` fn.